presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -892,6 +894,18 @@ public HiveClientConfig setAssumeCanonicalPartitionKeys(boolean assumeCanonicalP
         return this;
     }
 
+    public DataSize getMaxLineLength()
+    {
+        return maxLineLength;
+    }
+
+    @Config("mapreduce.input.linerecordreader.line.maxlength")
```

Comment:
This is the original config name in org.apache.hadoop.mapreduce.lib.input.LineRecordReader. @dain do you think it's ok to keep using it?

Reply:
It should follow the convention of the other file formats (e.g., `hive.orc.max-merge-distance`) so something like `hive.text.max-line-length`

Reply:
@dain would it cause confusion if it's different than the original config  in org.apache.hadoop.mapreduce.lib.input.LineRecordReader?

Reply:
I don't think so.  None of the configuration properties use the Hadoop names.  Instead, I think it might be more confusing to have one property that is not following the existing convention.

Reply:
Let's change the config name so that it's clear this is truncation. How about `hive.text.line-truncation-limit`.